clickplc/driver.py

```diff
@@ -389,3 +435,54 @@ def _pack(value):
             await self.write_registers(address, payload, skip_encode=True)
         else:
             await self.write_register(address, _pack(data), skip_encode=True)
+
+    def _load_tags(self, tag_filepath: str) -> dict:
+        """Load tags from file path.
+
+        This tag file is optional but is needed to identify the appropriate variable names,
+        and modbus addresses for tags in use on the PLC.
+
+        """
+        if not tag_filepath:
+            return {}
+        with open(tag_filepath) as csv_file:
+            csv_data = csv_file.read().splitlines()
+        csv_data[0] = csv_data[0].lstrip('## ')
+        parsed = {
+            row['Nickname']: {
+                'address': {
+                    'start': int(row['Modbus Address']),
+                },
+                'id': row['Address'],
+                'comment': row['Address Comment'],
+                'type': self.data_types.get(
+                    row['Address'].rstrip(digits).lower()
+                ),
+            }
+            for row in csv.DictReader(csv_data)
+            if row['Nickname'] and not row['Nickname'].startswith("_")
+        }
+        for data in parsed.values():
+            if not data['comment']:
+                del data['comment']
+            if not data['type']:
+                raise TypeError(
+                    f"{data['id']} is an unsupported data type. Open a "
```

Comment:
Can you check supported python versions for the click and see if f-strings are part of it? I'm okay limiting our support to 3.6+ so we can edit setup.py if needed.

Reply:
Add a version bump to this PR too. Probably a minor bump (major.minor.revision).

Reply:
We need 3.6 for f-strings (the prior PR made that change). No bump because this is part of the 0.3.0 changes that have not been released yet